django/db/backends/mysql/base.py

```diff
@@ -360,6 +360,8 @@ def check_constraints(self, table_names=None):
                     )
 
     def is_usable(self):
+        if self.connection is None:
+            return False
```

Comment:
Yes, never mind re: my alternative suggestion. As written that of course wouldn't work. :)